integration/basic_test.go

```diff
@@ -14,12 +14,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestOrcaMiniBlueSky(t *testing.T) {
+func TestLlamaBlueSky(t *testing.T) {
```

Comment:
```suggestion
func TestBlueSky(t *testing.T) {
```

This name is not accurate